src/chef-mover/src/mover_server_admins_existing_users_read_permissions_callback.erl

```diff
@@ -44,7 +44,7 @@ migration_action(UserRecord, _AcctInfo) ->
 add_permission_to_existing_user_for_server_admins(BifrostSuperuserId, Username, ServerAdminsAuthzId, UserAuthzid, Permission) ->
     case mv_oc_chef_authz:add_ace_for_entity(BifrostSuperuserId, group,  ServerAdminsAuthzId, actor, UserAuthzid, Permission) of
 	{error, Error} ->
-	    lager:error("Failed to update " ++ Permission ++ " permission for user " ++ Username ++ " with error:"),
+	    lager:error("Failed to update " ++ binary_to_list(Permission) ++ " permission for user " ++ binary_to_list(Username) ++ " with error:"),
```

Comment:
Yeah, definitely a better approach, but since this is tested, let's just use it.